### ISSUE TITLE
Examples simply don't work

### ISSUE THREAD

**Issue by hrqmonteiro** [nvimtools/hydra.nvim#38]
Tried setting the "Git" example:

Lazy:
![image](https://github.com/nvimtools/hydra.nvim/assets/17055027/aeeae6d8-eecb-47bc-831f-6c5eae182bf1)
![image](https://github.com/nvimtools/hydra.nvim/assets/17055027/fc85ae6a-0ede-4f2b-bc81-ef8630018fe9)
![image](https://github.com/nvimtools/hydra.nvim/assets/17055027/0627b879-c3d4-4b9d-961c-0e9597922984)

Hydra.lua:
![image](https://github.com/nvimtools/hydra.nvim/assets/17055027/b213f503-eaf2-4ade-977d-3feb4e0f57c2)

And when i press `<Leader>g` simply nothing happens:

https://github.com/nvimtools/hydra.nvim/assets/17055027/52d5d698-84f3-4d78-8d39-4d55c0e30f27

**Comment by miversen33** [comment]
Can you provide the version of neovim you are using?

**Comment by hrqmonteiro** [comment]
> Can you provide the version of neovim you are using?
> 
> Also, screenshots of configurations aren't useful, please provide the actual configurations you are using

```
NVIM v0.9.5
Build type: RelWithDebInfo
LuaJIT 2.1.1707061634
Compilation: /usr/bin/cc  -Og -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wvla -Wdouble-promotion -Wmissing-noreturn -Wmissing-format-attribute -Wmissing-prototypes -fno-common -Wno-unused-result -Wimplicit-fallthrough -fdiagnostics-color=auto -fstack-protector-strong -DUNIT_TESTING -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -I/usr/include/luajit-5_1-2.1 -I/usr/include -I/usr/include/lua5.1 -I/home/abuild/rpmbuild/BUILD/neovim-0.9.5/build/src/nvim/auto -I/home/abuild/rpmbuild/BUILD/neovim-0.9.5/build/include -I/home/abuild/rpmbuild/BUILD/neovim-0.9.5/build/cmake.config -I/home/abuild/rpmbuild/BUILD/neovim-0.9.5/src -I/usr/include -I/usr/include -I/usr/include -I/usr/include -I/usr/include -I/usr/include -I/usr/include

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"

Run :checkhealth for more info
```

My config with only Lazy, Neogit, Gitsigns and Hydra:

https://github.com/hrqmonteiro/dotfiles/tree/main/dot_config/nvim

**Comment by hrqmonteiro** [comment]
I erased my config and started one from scratch, i just have that Hydra on it, basically

Same thing happening, i press <Leader>g and it doesn't open:


https://github.com/nvimtools/hydra.nvim/assets/17055027/511b34f6-5bf4-48dd-9250-4ac26db1cff1

**Comment by matu3ba** [comment]
General tips for regressions:
- use as few dependencies as possible
- if more dependencies and/or arch dependent: mandatory ci testing + invoking bot to paste last known good commit combinations of plugins (quick notification as fast problem reduction to when problem might have been introduced)
- if ci not wanted, keep commits of neovim + plugins for manual checks and/or update the wiki entries accordingly

**Comment by miversen33** [comment]
@hrqmonteiro Are you pressing "the leader key" before pressing "g" in your video?

Your config works just fine if I set the leader to "space" (`vim.g.mapleader=" "`) though it appears that the default leader key in vim is `\`.

Looking at your video it appears you are just pressing the `g` button repeatedly.

I tested your config on both the most recent nvim 0.9.5 stable _and_ nvim 0.10 nightly and it worked as expected on both.

Hydra uses a leader (the head) and a chord (the body). Try setting a leader key and seeing if it works
